ocaml-lsp-server/src/document.mli

```diff
@@ -45,6 +45,10 @@ val source : t -> Msource.t
 
 val with_pipeline_exn : t -> (Mpipeline.t -> 'a) -> 'a Fiber.t
 
+(** [get_parsetree_exn doc] returns a fiber with [doc]'s parsetree. Keep in
+    mind, this function is run on the merlin thread *)
+val get_parsetree_exn : t -> Mreader.parsetree Fiber.t
```

Comment:
This function is one line long, only used once, and doesn't offer much abstraction. I'd just inline it in the caller.

Reply:
This function is still useless